### ISSUE TITLE
Validation AUC as metric [tune]

### ISSUE THREAD

**Issue by gagasth** [ray-project/ray#13999]
Is there a way to make the validation AUC the metric for tune.run? With AUC set as the metric, the hyperparameter tuning is leading to overfitting. Could you point me to an example where this is being done? I am trying to tune a neural network.
### Describe your feature request

For feature requests or questions, post on our Discussion page instead: https://discuss.ray.io/

**Comment by krfricke** [comment]
Hi @gagasth, this mostly depends on your training mechanism. Tune can work with any kind of metric, but you'll have to define this in your trainable function.

Can you share your training code?

By the way, this kind of question is best asked on discuss.ray.io, as other users can chime in and might be able to help. Can you post the question there (with some additional context)?
